src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const disc3 = 20;
+  const disc7 = 50;
```

Comment:
It's good practice to make variables for reusable data, but let's give it more meaningful names.
It's not clear what does 'disc3' or 'disc7' means. A name like 'discount3' would be more understandable.